pkg/queue_handler/poller.go

```diff
@@ -0,0 +1,94 @@
+// Package queue_handler updates quota on a Store from messages arriving on
+// a queue.
+package queue_handler
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/treeverse/terminus/pkg/store"
+)
+
+// Poll repeatedly long-polls on client, and writes SQS messages to outCh,
+// until ctx is cancelled.
+func Poll(ctx context.Context, l *log.Logger, client sqs.SQS, queueUrl string, keyPattern *regexp.Regexp, keyReplace string, s store.Store) {
+	for {
+		in := &sqs.ReceiveMessageInput{
+			// TODO(ariels): AttributeNames
```

Comment:
If this TODO stays here, please elaborate

Reply:
Done!